roles/quay/defaults/main.yml

```diff
@@ -1,9 +1,7 @@
 ---
-certificate_path: '{{ tmp_dir }}/certs'
-quay_password: '{{ lookup("password", tmp_dir + "/quay.password chars=ascii_letters,digits") }}'
```

Comment:
@jharmison-redhat did you mean to delete this? It's is still referenced in the quay.yml.j2 template and that is throwing an error

`TASK [quay : Subscribe to Quay Enterprise] ******************************************************************************************************************************
task path: /app/roles/quay/tasks/main.yml:2
Wednesday 23 September 2020  18:07:51 +0000 (0:00:00.138)       0:00:01.871 ***
File lookup using /app/roles/quay/templates/quay.yml.j2 as file
fatal: [localhost]: FAILED! => {}

MSG:

The task includes an option with an undefined variable. The error was: 'quay_password' is undefined

The error appears to be in '/app/roles/quay/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- name: Subscribe to Quay Enterprise
  ^ here
`

Reply:
see 8dc9a38

The Quay Operator no longer respects admin credentials passed into the `QuayEcosystem` - it will always create `quay:password` by default and this should be either:
manually changed
changed after Quay is stood up